include/torch-mlir/Dialect/Torch/IR/GeneratedTorchOps.td

```diff
@@ -6743,6 +6743,31 @@ def Torch_Aten_AdaptiveAvgPool3dBackwardOp : Torch_Op<"aten._adaptive_avg_pool3d
   }];
 }
 
+def Torch_AtenAdaptiveMaxPool2dOp : Torch_Op<"aten.adaptive_max_pool2d", [
```

Comment:
Too bad we don't have the flexibility with the automatic generation to make the adaptive pooling ops 'inherit' from the same class.

Reply:
Yeah, I can do that if it makes more sense. Someone wanted this specific op lowered in [SHARK-Turbine#116](https://github.com/nod-ai/SHARK-Turbine/issues/116). 

I'm planning to refactor in the next patch I put up for this, but figured I would do this as a test run for the max ops and get support for this op ASAP while working on the rest.